proposed/extended-coding-style-guide.md

```diff
@@ -905,6 +905,68 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Compound boolean statements
+
+A compound boolean expression MAY be split across multiple lines. When splitted, 
+there MUST be only one boolean expression per line, or an other compound boolean
+expression that MUST be enclosed by parentheses. The logical operator MUST be placed 
+at the beginning of each line. 
+
+~~~php
+    return $firstBoolean
+        && ($secondBoolean || $thirdBoolean)
```

Comment:
Aren't there *two* boolean expressions per line?

Reply:
Yes but:

> there MUST be only one boolean expression per line, **or an other compound boolean expression** that MUST be enclosed by parentheses.

I don't want to force the splitting for every single element, that example is more than readable, and it emphasize that the inner compound element is different from the others (it uses `||` instead of `&&`)

Reply:
OK.